tests/coalaCITest.py

```diff
@@ -33,9 +33,7 @@ def test_nonexistent(self, debug=False):
         retval, stdout, stderr = execute_coala(
             coala.main, 'coala', '--non-interactive', '-c', 'nonex', 'test')
```

Comment:
i bump this. we are not going to merge this if the tests are not in the corresponding commit. i told you the problems that arise. if you revert, it will fuck up.

in each commit, for each line that is modified/added, its tests need modifying too, thats how you should have done it from the start and it wouldnt be coverage problems